src/createfeedback/CreateFeedback.test.js

```diff
@@ -0,0 +1,114 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { CreateFeedback } from './CreateFeedback';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import { Description } from '@mui/icons-material';
+
+describe('Create Feedback Initial State', () => {
+  test('Should have a link to go back with an arrow, a plus icon, Create New Feedback heading, feedback title, feedback title description, feedback text box, category title, category title description, dropdown menu, feedback detail heading, feedback detail description, feedback detail text box, add feedback button and cancel button', () => {
```

Comment:
Love the description 😄 you can use your discretion - if it's this long you don't _need_ to list them all out. You're the author! No question as to what's getting tested here